### PR TITLE
Update depth-guided stable diffusion example to diffusers 0.27.2

### LINKED ISSUE #5976: `controlnet` and `depth_guided_stable_diffusion` have conflicting dependencies
"diffusers>=0.12.1,<=0.21" for `depth_guided_stable_diffusion`
"diffusers==0.27.2" for `controlnet`

Either or both should be updated to resolve that conflict.

### PR DESCRIPTION
**Pull request by roym899**
### What
Closes #5976 

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5985?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/5985)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: scripts/lint.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
 from __future__ import annotations
 
 import argparse
-from glob import glob
 import os
 import re
 import sys
+from glob import glob
```
Comment (emilk) [inline scripts/lint.py:15]: Why wasn't this caught by CI? 🤔

Comment (Wumpf) [reply scripts/lint.py:15]: Looked into it and fixed it, see https://github.com/rerun-io/rerun/pull/5995

**Review by Wumpf — APPROVED** [review]
nice, thank you so much for going the extra mile and clean up the readme and entity names some more!

**Comment by Wumpf** [comment]
(doing a local run, then merging)

**Pull request merged**